### DISCUSSION TITLE
Events are not sent to Sentry after upgrading sentry-laravel from 2.9 to 4.2 and laravel from 9 to 10

### DISCUSSION THREAD

**Discussion by agilan-selvam-zoomrx** [getsentry/sentry-laravel#847] [Q&A] [answered]
Hey Everyone,

I have an application which runs on Laravel. It has laravel 9.x and sentry-laravel 2.9. I have upgraded it recently to 10.x and sentry-laravel to 4.2. After upgrading am not able to send any events to sentry and php artisan sentry:test also fails with below error.

Error:

> DSN discovered from Laravel config or `.env` file!
> Sending test event...
> There was an error sending the event.
> Please check if your DSN is set properly in your `.env` as `SENTRY_LARAVEL_DSN` or in your config file `config/sentry.php`.

But if i switch back to previous version, am able to send events to sentry. Am using self hosted sentry and its version is 9.1.2

Can someone help me out to solve this issue.

Thanks in advance.

**Comment by stayallive** [comment]
Unfortunately version 4.x doesn't work for self-hosted instances that old.

See the release notes: https://github.com/getsentry/sentry-laravel/releases/tag/4.0.0 (the first point is relevant for you).

You can stay on `sentry-laravel` 3.x which should also be compatible with Laravel 9.

**Reply by agilan-selvam-zoomrx** [reply]
@stayallive  Thanks alot for the quick response. This helps.

But am facing the same error in 3.2.0 as well for Laravel 10. Am I missing anything? I havent changed anything in the application code, except updating the version and composer install.

**Reply by cleptric** [reply, marked as answer]
Please try 3.8.2. There was an issue with some cURL version we released a fix for.

**Reply by agilan-selvam-zoomrx** [reply]
@cleptric Thank you so much for the solution. After changed to this version, exceptions or errors are getting captured in sentry, but still the sentry:test command is throwing the same error. Not sure why it is not working.

Once again thanks for the suggesting. This works for now.
